mqtt-bridge/src/main/java/io/strimzi/kafka/bridge/mqtt/core/MqttServerHandler.java

```diff
@@ -0,0 +1,82 @@
+package io.strimzi.kafka.bridge.mqtt.core;
+
+
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.SimpleChannelInboundHandler;
+import io.netty.handler.codec.mqtt.MqttMessage;
+import io.netty.handler.codec.mqtt.MqttMessageBuilders;
+import io.netty.handler.codec.mqtt.MqttMessageType;
+import io.netty.handler.codec.mqtt.MqttPublishMessage;
+import io.netty.handler.codec.mqtt.MqttConnectReturnCode;
+import io.netty.handler.codec.mqtt.MqttConnAckMessage;
+
+/**
+ * Represents a SimpleChannelInboundHandler. The MqttServerHandler is responsible for: <br>
+ * - listen to client connections;<br>
+ * - listen to incoming messages; <br>
+ *
+ * @see io.netty.channel.SimpleChannelInboundHandler
+ */
+public class MqttServerHandler extends SimpleChannelInboundHandler<Object> {
```

Comment:
I guess you can use `MqttMessage` directly (as concrete type) and not casting the `Object` msg later in the `channelRead`, so something like ...

```java
public class MqttServerHandler extends SimpleChannelInboundHandler<MqttMessage>
```